ngx_http_websocket_stat_module.c

```diff
@@ -449,6 +351,17 @@ ws_packet_size(ngx_http_request_t *r, void *data)
     return (char *)buff;
 }
 
+const char *
+ws_message_size(ngx_http_request_t *r, void *data)
+{
+    template_ctx_s *ctx = data;
+    ngx_frame_counter_t *frame_cntr = &ctx->ws_ctx->frame_counter;
+    if (!ctx || !frame_cntr)
```

Comment:
UB: you already dereferenced `ctx`, so it cannot be `null` here

Reply:
Thanks for that! Will specify types of the arguments explicitly